### DISCUSSION TITLE
GitHub issue overhall

### DISCUSSION THREAD

**Discussion by josphh** [nix-community/stylix#141] [Ideas]
As Stylix is growing larger, maybe it would be wise to write up some custom issue templates, and some unique labels? This will help with troubleshooting/bug-hunting, as well as allowing users to possibly find fixes to problems themselves, or prevent duplicates. Labels could also help with sorting between stylix-breaking critical issues, and cosmetic ones.
I'll be happy to write up issue templates if needed.

**Comment by SomeGuyNamedMay** [comment]
this would absolutly be helpfull, though stylix's internals change fairly rapidly atm so this might be better as a long term goal

**Comment by danth** [comment]
Some basic templates would be nice, but we should avoid having anything too restrictive which might put people off from opening issues.
